### ISSUE TITLE
Difficulty descending stairs

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5933]
**Preferred term label:** Difficulty descending stairs

**Synonyms** Difficulty walking down stairs

**Definition (free text, please give PubMed ID)** Reduced ability to descend stairs.

**Parent term (use hpo.jax.org/app)** Functional motor deficit HP:0004302

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
